DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Http Client bundle.
+ *
+ * (c) David Buchmann <mail@davidbu.ch>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Http\ClientBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+
+/**
+ * This class contains the configuration information for the bundle
+ *
+ * This information is solely responsible for how the different configuration
+ * sections are normalized, and merged.
+ *
+ * @author David Buchmann <mail@davidbu.ch>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritDoc}
```

Comment:
We tend to use `{@inheritdoc}`. It doesn't matter IMO, but should be consistent.